module/userProjects/helper.js

```diff
@@ -1630,6 +1630,8 @@ module.exports = class UserProjectsHelper {
 				let projectData = await _projectData(data)
 				if (projectData && projectData.success == true) {
 					createProject = _.merge(createProject, projectData.data)
+				} else {
+					return reject(projectData)
```

Comment:
@MallanagoudaB why this change required for add API category validation?

Reply:
Because if the id is not there in DB this will handle it with below message.
{
    "status": 400,
    "message": "Not found category"
}